Core/DefInjected/ThingDef/Buildings_Misc.xml

```diff
@@ -2,105 +2,105 @@
-  <OrbitalTradeBeacon.description>TODO</OrbitalTradeBeacon.description>
+  <OrbitalTradeBeacon.description>Vereist om handel met ruimtehandelaren te drijven. Enkel voorwerpen binnen bereik van een baken kunnen worden verhandeld. Kan binnenshuis worden geplaatst.</OrbitalTradeBeacon.description>
   
   <!-- EN: party spot -->
-  <PartySpot.label>TODO</PartySpot.label>
+  <PartySpot.label>feestplek</PartySpot.label>
   <!-- EN: Designates a spot for throwing parties. -->
-  <PartySpot.description>TODO</PartySpot.description>
+  <PartySpot.description>Wijst een plek aan waar feestjes kunnen worden gegeven.</PartySpot.description>
   
   <!-- EN: pod launcher -->
-  <PodLauncher.label>TODO</PodLauncher.label>
+  <PodLauncher.label>capsule lanceerbasis</PodLauncher.label>
   <!-- EN: A fueling port for launching one transport pod. Pod launchers can launch as a group - but the launchers must be placed adjacent to each other. -->
-  <PodLauncher.description>TODO</PodLauncher.description>
+  <PodLauncher.description>Een brandstofdepot waarmee een enkele transportcapsule kan worden gelanceerd. Capsules kunnen in groepen worden gelanceerd, maar dan moeten de lanceerbases wel tegen elkaar aan worden gebouwd.</PodLauncher.description>
   
   <!-- EN: sarcophagus -->
-  <Sarcophagus.label>TODO</Sarcophagus.label>
+  <Sarcophagus.label>sarcofaag</Sarcophagus.label>
   <!-- EN: An ornamented burial receptacle for the honored dead. -->
-  <Sarcophagus.description>TODO</Sarcophagus.description>
+  <Sarcophagus.description>Een bewerkte grafkist bedoeld voor vereerde doden.</Sarcophagus.description>
   
   <!-- EN: grand stele -->
-  <SteleGrand.label>TODO</SteleGrand.label>
+  <SteleGrand.label>indruk-\nwekkende zuil</SteleGrand.label>
   <!-- EN: A towering slab with extensive engravings on the sides. Steles have been used since ancient times to memorialize individuals, battles, and other important events. -->
-  <SteleGrand.description>TODO</SteleGrand.description>
+  <SteleGrand.description>Een enorm stuk steen waarin gedetailleerd graveerwerk is aangebracht. Zuilen worden al eeuwenlang gebruikt om individuen, veldslagen en andere belangrijke gebeurtenissen vast te leggen.</SteleGrand.description>
   
   <!-- EN: large stele -->
-  <SteleLarge.label>TODO</SteleLarge.label>
+  <SteleLarge.label>grote zuil</SteleLarge.label>
   <!-- EN: A tall, thick slab with engravings on the sides. Steles have been used since ancient times to memorialize individuals, battles, and other important events. -->
-  <SteleLarge.description>TODO</SteleLarge.description>
+  <SteleLarge.description>Een groot stuk steen waarin graveerwerk is aangebracht. Zuilen worden al eeuwenlang gebruikt om individuen, veldslagen en andere belangrijke gebeurtenissen vast te leggen.</SteleLarge.description>
   
   <!-- EN: tool cabinet -->
-  <ToolCabinet.label>TODO</ToolCabinet.label>
+  <ToolCabinet.label>gereed-\nschapskast</ToolCabinet.label>
```

Comment:
Is dit een typo of wordt dit door Rimworld begrepen dat er een LF in de label moet zitten?

Reply:
Dat laatste is het geval. Zonder newlines worden namen op de knoppen niet goed gebroken, en dat komt de leesbaarheid niet ten goede. In mijn tests ziet het er goed uit:

![image](https://user-images.githubusercontent.com/119742/98482086-1fdebc00-21ff-11eb-8517-d5e753bfe3b2.png)

Reply:
Cool, ik wist niet dat dit kon :)